journal/journal.go

```diff
@@ -1,4 +1,3 @@
-// Copyright 2015 RedHat, Inc.
```

Comment:
Sorry, the diff is pretty broken here. These are really just file moves:
`journal/read.go` -> `journal/journal.go`
`journal/journal.go` -> `sdjournal/journal.go`